python/ray/tune/log_sync.py

```diff
@@ -55,7 +55,7 @@ def set_worker_ip(self, worker_ip):
 
     def _check_valid_worker_ip(self):
         if not self.worker_ip:
-            logger.info("Worker ip unknown, skipping log sync for {}".format(
+            logger.debug("Worker ip unknown, skipping log sync for {}".format(
```

Comment:
Should this be `logger.warning` or something? It might be hard for anyone to catch that the log sync fails if this is only shown on debug level.

Reply:
I took this out because it keeps getting printed even on single node where it isn't applicable, causing a bit of confusion. I think actually this error message isn't very informative, as it doesn't provide anything for the user to act on, but I'm not sure how to improve it (mainly because I haven't seen many failure modes)...